cluster-autoscaler/cloudprovider/azure/azure_manager.go

```diff
@@ -315,7 +315,7 @@ func (m *AzureManager) getFilteredScaleSets(filter []labelAutoDiscoveryConfig) (
 		}
 		spec := &dynamic.NodeGroupSpec{
 			Name:               *scaleSet.Name,
-			MinSize:            1,
+			MinSize:            0,
```

Comment:
Would suggest default as 1. Can remove code that ignores vmss creation if min-size not provided.